e-library/src/Components/forms/createbook.jsx

```diff
@@ -1,29 +1,69 @@
-import React from "react";
+import React, { useState } from "react";
+import { useNavigate } from 'react-router-dom';
+import axios from "../api/axios";
+import formData from 'form-data'
+import { token } from './input';
 
-const CreateBook = () => {
-    return(
-        <form >
-        <h2>Create your book</h2>
-        <div className="input-box">
-            <input type="text" placeholder="Title"/>
-        </div>
-        <div className="input-box">
-           <textarea placeholder="Description" rows="4" columns="50"></textarea>
-        </div>
-        <div className="input-box">
-            <input type="text" placeholder="Author"/>
-        </div>
-        <div className="input-box">
-            <input type="text" placeholder="Price"/>
-        </div>
-        <div className="input-box">
-            <input type="file"/>
-        </div>
-        <div className="input-box">
-            <input type="submit" value="Create your book"/>
-        </div>
-    </form>
-    )
+const CreateBook = () => {;
+    const [title, setTitle] = useState('');
+    const [price, setPrice] = useState('')
+    const [author, setAuthor] = useState('');
+    const [fileUrl, setFileUrl] = useState(null)
+    const [description, setDescription] = useState('')
```

Comment:
instead of having multiple state instances. why is everything not in an object using one state?